### PR TITLE
[CCTRI-1992] Tweak README & User-Agent to have SecureX Threat Response wording

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Config:
 
     SECRET_KEY = os.environ.get('SECRET_KEY', None)
 
-    USER_AGENT = ('Cisco Threat Response Integrations '
+    USER_AGENT = ('SecureX Threat Response Integrations '
```
Comment (GeorgiyII) [inline config.py:12]: Are you sure that it should be so and not `Cisco SecureX Threat Response Integrations`?

**Review by GeorgiyII — COMMENTED** [review]
README.md line 148 and 149 missed

File: config.py (hunk 1 above)
Comment (AlinaBlahun) [reply config.py:12]: According to Vika's message, it's "Cisco Threat Response to SecureX Threat Response". So I guess it's done correctly.

Comment (GeorgiyII) [reply config.py:12]: No problem))

**Review by GeorgiyII — APPROVED** [review]

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**